docs/lago-tutorial/README.rst

```diff
@@ -0,0 +1,242 @@
+although you will get the ability to create your own custom environment.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Install Lago - follow
+`this <../README.html#installation>`__ tutorial
+for more information
+
+The scenario
+^^^^^^^^^^^^
+
+Create a Lago environment which will consist of three virtual machines
+that will host Jenkins infrastructure.
+
+The VMs
+^^^^^^^
+
+-  "vm0-server" - Jenkins server
+-  "vm1-slave" - Jenkins slave
+-  "vm2-slave" - Jenkins slave
+
+The network
+^^^^^^^^^^^
+
+All the VMs will connect to the same virtual network.
+Lago allows for a creation of 10 different lans ranging from:
+192.168.200.x to 192.168.209.x
+The subnet will be automatically assigend to the virtual network
+bridge.
+
+Creating the working directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You should copy 'lago/docs/lago-tutorial' from this repository to your machine.
+
+The components  
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this directory you will find the following:
+
+-  init.json – This is the file which describes the structure of our
+   environment: the specification of the vms, networks, the path and
+   name of the deployment scripts for each vm.
+
+-  scenarios - This directory will contain default configuration files
+   for common enviornments - you are welcome
+   to conribute your on files :)
+
+-  deployment-scripts – This directory contains the deployment scripts
+   for each vm (in our case a script that will
+   install the Jenkins server)
+
+Editing init.json
+^^^^^^^^^^^^^^^^^
+
+The init.json is where we need to describe our environment e.g vms and
+networks, so Lago can create them for us.
+
+//TODO: elaborate about the different features and values that can be
+specified within the file.
+
+Creating the environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can use two different approches:
+
+Manual Configurations:
+
+::
+
+    lago init lago-work-dir init.json
+
+(This should be invoked from: /lago-tutorial/scenarios/jenkins)
+
+-  The directory /lago-work-dir will contain the files of our new Lago
+   environment.
+   This directory shouldn't exist before invoking lago init.
+   
+-  --template-repo-path, for specifing the path to the template-repo
+   file. (The templates repository is the place from which the base “qcow2” virtual disks will be copied from).
+   By default Lago will use the file from: http://templates.ovirt.org/repo/repo.metadata\
+   
+-  init.json, the name of the file which describes our environment.
+
+Auto configurations
+
+::
+
+    lago init
+
+This command will use the following default configurations:
+
+-  The command will search for a file named "LagoInitFile" that will be
+   used as the init file.
+-  The workdir will be named ".lago"
+-  As mentioned above, The default repo will be used.
+
+Note: Lago will copy the deployment scripts into the new environment, and set up the following environment variables:
+
+- $LAGO_PREFIX_PATH = FULL_PATH/.lago/current
```

Comment:
I'd use an example instead of 'FULL_PATH', something like:
```
... and set up the following variables (for example, if you were at /home/myuser/lago dir):
- $LAGO_PREFIX_PATH = /home/myuser/lago/.lago/current
```